test/helper/utils.js

```diff
@@ -0,0 +1,10 @@
+const chai = require('chai');
+const app = require('../../index')
+
+module.exports.getCsrfTokenHtml = function getCsrfToken(request) {
+  return request.res.text.split('<input type="hidden" name="_csrf" value="')[1].split('">')[0];
+}
+
+module.exports.getCsrfTokenCookie = function getCsrfToken(request) {
```

Comment:
Cette fonction récupère tous les cookies, pas juste le CSRF, non ? On pourrait la faire récupérer juste le CSRF, pour éviter les effets de bords imprévus ? 
Ou sinon si c'est plus simple, on la renomme "getCookiesFromRequest" un truc comme ca

```suggestion
module.exports.getCsrfTokenFromCookie = function getCsrfTokenFromCookie(request) {
```
(attention renommer partout)